tests/files.ts

```diff
@@ -35,6 +35,9 @@ test.serial(
 		const { body } = t.context;
 		state.set("fileId", body.id);
 		t.is(body.versions.length, 1, "File should have 1 version, the initial one");
+		unstableValues.add(body.id);
+		unstableValues.add(body.versions.map((x: any) => x.created_at));
+		unstableValues.add("https://vrchat.com/api/1/file/"+body.id);
```

Comment:
You don't need this line as `unstableValues.add(body.id)` will already redact the value that changes.